Polyhedron/doc/Polyhedron/CGAL/Polyhedron_3.h

```diff
@@ -1096,6 +1096,8 @@ class Polyhedron_3 {
   */
   Vertex_iterator vertices_end();
 
+  unspecified_type vertex_handles();
```

Comment:
You should add Doxygen based docs to all added methods. See as it is done for other methods around. Basically it is an explanation within the structure: `\* comment *\` .